snapcraft.yaml

```diff
@@ -422,9 +426,25 @@ parts:
           build_edk2 \
             "${CRAFT_PART_INSTALL}/share/qemu/OVMF_CODE.4MB.CSM.fd" \
             "${CRAFT_PART_INSTALL}/share/qemu/OVMF_VARS.4MB.CSM.fd" \
+            RELEASE \
             -DFD_SIZE_4MB \
             -DCSM_ENABLE=TRUE
       fi
+
+      # Debug firmware (4MB, debug mode)
```

Comment:
Can we add some reference in the comments here to the new config option in lxd that will use thus firmware so its clear the two are related when doing a cross repo code search.

Reply:
Agree, have added ;-)